packages/sync/tsconfig.build.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "../../tsconfig/isomorphic.json",
```

Comment:
It would be nice if this package could be isomorphic, but I don't think its deps really permit it.
```suggestion
  "extends": "../../tsconfig/node.json",
```

Reply:
good catch